modules/islandora_scholar_embargo/includes/admin.form.inc

```diff
@@ -21,7 +21,9 @@ function islandora_embargo_admin(array $form, array &$form_state) {
   $options = islandora_get_content_models();
   $selected = variable_get('islandora_embargo_content_models', array('ir:citationCModel', 'ir:thesisCModel'));
   foreach ($selected as $cmodel) {
-    $options = array($cmodel => $options[$cmodel]) + $options;
+    if (isset($options[$cmodel])) {
+      $options = array($cmodel => $options[$cmodel]) + $options;
+    }
```

Comment:
Yeah, it wasn't so much the `isset()`, but the whole: "options = one of it's items + the rest of it"...  Which would only have the affect of (re)ordering the array...?